src/components/outlier-analysis-table/OutlierAnalysisTable.js

```diff
@@ -15,51 +15,88 @@ import DownloadAs from '../../components/download-as/DownloadAs'
 import i18n from '@dhis2/d2-i18n'
 import { apiConf } from '../../server.conf'
 import cssPageStyles from '../../pages/Page.module.css'
-import jsPageStyles from '../../pages/PageStyles'
+// import jsPageStyles from '../../pages/PageStyles'
```

Comment:
Should we just remove this line or is there a reason to leave it as a comment?  Same with several other lines below

Reply:
I should have explained what Hendrik and I talked about in the call actually. So Hendrik will move all the commented code to a separate branch, it's functionality that he created but that won't work with the current backend (if I'm not mistaken). In the future we should be able to merge that as well, once the functionality is there.

Reply:
As discussed with @ismay I will leave the comments in, to avoid merge conflicts later. They will be uncommented fairly soon.